### PR TITLE
Move scope/contributing guidelines to README

### PR DESCRIPTION
**Pull request by dstansby**
~~This is what the scope of this repository has been in my head for a while, so I thought it worth getting it down in the README and seeing if others agree.~~

I think it's worth moving these bits from CONTRIBUTING.md to the README for visibility.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,11 @@ Python packages with our recommended tooling set up and ready to go.
 > package template (e.g., [`scikit-hep`](https://github.com/scikit-hep/cookie)),
 > we recommend using their template instead of this one.
 
+## Scope and contributing
+
+This project is written by members of ARC for members of ARC, and is scoped to be a helpful tool for the projects we work on in ARC.
+As such contributions to this repository are only open to members ARC, but we welcome (and encourage!) suggestions for improvements or bug fixes from anyone.
```
Comment (samcunliffe) [inline README.md]: There already is a CONTRIBUTING.md with

> This template and our [recommendation pages](https://github-pages.arc.ucl.ac.uk/python-tooling) were made by [research software engineers](https://society-rse.org/about/history) at [UCL's Centre for Advanced Research Computing](https://ucl.ac.uk/arc). We made it with research software projects in mind, but whoever you are, we hope you find this useful!

Does that make the scope clear enough?

**Review by samcunliffe — CHANGES_REQUESTED** [review]
We definitely shouldn't have separate contributing guidelines in README.

**Comment by dstansby** [comment]
🤦 my bad for not spotting that, but I think worth moving to the README for visibility?

**Comment by dstansby** [comment]
Have updated the description at the top to explain the new commit.

**Comment by samcunliffe** [comment]
> 🤦 my bad for not spotting that, but I think worth moving to the README for visibility?

I think the other way around. People who want to contribute should expect the contribution guidelines to be in the CONTRIBUTING.md and should not have to also read a section of the README.

README for users, CONTRIBUTING for contributors.

**Comment by samcunliffe** [comment]
I also don't mind having a quite permissive scope. PRs and issues from everyone but PRs reviewed by ARC or associate member of ARC (which is what I was going for [when I wrote the contribution guidelines](https://github.com/UCL-ARC/python-tooling/pull/278)).

We keep it "with research software projects in mind".

**Comment by paddyroddy** [comment]
> I also don't mind having a quite permissive scope. PRs and issues from everyone but PRs reviewed by ARC or associate member of ARC (which is what I was going for [when I wrote the contribution guidelines](https://github.com/UCL-ARC/python-tooling/pull/278)).

I agree with this sentiment

File: README.md [hunk 2]
```diff
@@ -17,12 +17,24 @@ Python packages with our recommended tooling set up and ready to go.
 > package template (e.g., [`scikit-hep`](https://github.com/scikit-hep/cookie)),
 > we recommend using their template instead of this one.
 
+## Scope and contributing
+
+This project was created by [research software engineers] at [UCL's Centre for Advanced Research Computing][UCL ARC].
+We made it with our own research software projects in mind - we don't explicitly support external users, but whoever you are, we hope you might find it useful!
+
+As such, contributions to this repository are only open to members ARC - if you're a member of ARC, see [CONTRIBUTING.md] for more info on how to contribute.
```
Comment (paddyroddy) [inline README.md]: I don't like this at all

**Review by paddyroddy — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (dstansby) [reply README.md]: Why?

Comment (samcunliffe) [reply README.md]: If someone outside ARC has the impetus to write a PR to try and improve our tool, I will be very willing to review that.

**Comment by dstansby** [comment]
> I think the other way around. People who want to contribute should expect the contribution guidelines to be in the CONTRIBUTING.md and should not have to also read a section of the README.

I agree for the details of how to contribute, but I think scope is out of scope (😆) for CONTRIBUTING.md, and contributor scope (ie. am I allowed to contribute to this repository?) should be more easily findable with a link to CONTRIBUTING.md because (as far as I can tell?) GitHub doesn't obviously expose CONTRIBUTING.md anywhere [on the repo home page](https://github.com/UCL-ARC/python-tooling).

File: README.md (hunk 2 above)
Comment (dstansby) [reply README.md]: It was more a question for @paddyroddy, since simply writing

> I don't like this at all

is not a helpful review 😉

Comment (samcunliffe) [reply README.md]: Also this feels unnecessarily unwelcomming. If we have floods of contributions, maybe let's think again. But the odd PR from --say-- a UCL student or external researcher or a _chof cough_ RSE from Manchester, is very welcome.

**Comment by dstansby** [comment]
I have reverted this back to open contribution, given we have active maintainers happy to review external stuff 👍

I still think worth this going in the README so there's an up-front "contributors welcome" message on https://github.com/UCL-ARC/python-tooling?

File: README.md [hunk 3]
```diff
@@ -17,12 +17,26 @@ Python packages with our recommended tooling set up and ready to go.
 > package template (e.g., [`scikit-hep`](https://github.com/scikit-hep/cookie)),
 > we recommend using their template instead of this one.
 
+## Scope and contributing
+
+This project was created by [research software engineers] at [UCL's Centre for Advanced Research Computing][UCL ARC].
+We made it with our own research software projects in mind - we don't explicitly support external users, but whoever you are, we hope you might find it useful!
```
Comment (samcunliffe) [inline README.md:23]: I also don't really like "we don't explicitly support external users".

I'm biased to how I had it, but I did spend effort thinking carefully about how it was worded.

File: README.md [hunk 4]
```diff
@@ -17,12 +17,26 @@ Python packages with our recommended tooling set up and ready to go.
...
+
+This repository is open to contributions from all, but they will need be reviewed by a member or associate member of ARC.
+We might be slow to approve new tool suggestions (since we'll probably want to discuss them first) but don't let that put you off creating an issue or opening a pull request.
+See [CONTRIBUTING.md] for more details on how to contribute.
+
+We also accept (and encourage!) suggestions and bug reports from anyone.
+Use the [discussions tab] of this repository to ask questions or [open an issue] if something seems like it's broken.
```
Comment (samcunliffe) [inline README.md:30]: Ditto with how I worded previously...

> We are actively encouraging users to ask questions and start discussions in the
[discussions tab] of this repository. Does something seem like it's broken?
Please go ahead and [open an issue]!

File: README.md (hunk 2 above)
Comment (paddyroddy) [reply README.md]: @dstansby I thought it was obvious in context, but yes, basically what @samcunliffe said.

**Review by samcunliffe — CHANGES_REQUESTED** [review]
Sorry, I really don't agree to put contribution guidelines in README.

If the problem is that GH doesn't advertise CONTRIBUTING like it does with the CoC, I'd propose to simply link from the README...

```md

## Contributions

... are welcome. Please see our [./CONTRIBUTING.md] file for details.

```

File: README.md (hunk 3 above)
Comment (paddyroddy) [reply README.md:23]: I like the current one. Whatever happens this should be the "groups" view, not one individuals.

**Comment by dstansby** [comment]
I'm going to close this as I seem to have got a bit too keen with a PR without checking previous history/opening an issue first. Sorry for the noise/angst/wasting your time, but thanks for reviewing

**Pull request closed without merging**